<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zona Interior</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>je.png</normaloff>je.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>371</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tribunal Regional Eleitoral do Estado da Bahia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>97</width>
      <height>97</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>je.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inconsolata Semi Expanded Black</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Justiça Eleitoral</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBut">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>520</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Computador Localizado em Cartórios do Interior</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
